### ISSUE TITLE
Example data for training Success Detector

### ISSUE THREAD

**Issue by pmj110119** [zhouzypaul/auto_eval#1]
Thank you for open-sourcing this fascinating work!

I would like to know if you can provide a Success Detector training data as an example to help us better understand data collection techniques. The classification model I trained using my own collected success/failure images performed poorly.

**Comment by zhouzypaul** [comment]
Hi, 

Yes, I can share some training data as an example -- though I'm not sure how useful it will be. 

What task are you trying to train? What is your formulated EQA question for Paligemma? And what is your trained classifier accuracy? There are some additional info for training the classifier in the appendix of the paper in case you haven't seen it.

**Comment by pmj110119** [comment]
Thanks!
 It would be great if you could provide data for any one of the tasks: "drawer", "sink", or "cloth".


In my experiments, the VLM started to overfit after training for just 5/80 epochs on 1,000 images (the eval_loss increased significantly, far exceeding the initial training loss). I'd like to take a look at the training curves corresponding to your data.
![Image](https://github.com/user-attachments/assets/dfaf11f5-ff8d-43d4-9b4c-d738e4c68760)

**Comment by zhouzypaul** [comment]
Sure, here is the data for the sink task: https://huggingface.co/datasets/zhouzypaul/sink-classifier-data/tree/main

Hope that helps! You may also try to increase the batch size. 

Closing for now, but feel free to re-open if you run into more issues. There is also a discord channel where we can discuss more if that makes things easier.
